include_dir/src/lib.rs

```diff
@@ -56,21 +49,5 @@ pub use dir::Dir;
 pub use file::File;
 
 #[doc(hidden)]
-pub use include_dir_impl::*;
-
-#[macro_export]
-#[doc(hidden)]
-/// Hack used by `include_dir_impl` which can't access `$crate`.
-macro_rules! __include_dir_use_everything {
-    () => {
-        pub use $crate::*;
-    };
-}
-
-proc_macro_expr_decl! {
-    include_dir! => include_dir_impl
-}
-
-/// Example the output generated when running `include_dir!()` on itself.
-#[cfg(feature = "example-output")]
-pub static GENERATED_EXAMPLE: Dir = include_dir!(".");
```

Comment:
I'm on the fence about removing the generated example. I really like when other crates which use macros to generate code will show an example of what gets generated as part of their docs (e.g. [error-chain](https://docs.rs/error-chain/0.12.0/error_chain/example_generated/index.html)).